commands/get/resources/getter_test.go

```diff
@@ -0,0 +1,155 @@
+			getter.On("GetRemoteResources", destination, constants.DefaultCertificationsFolder, dependentCertifications).Return(expectedError)
+		})
+		It("should return an error when it's unable to get remote standards", func() {
+			expectedError = errors.New("Remote standards error")
+			getter.On("GetLocalResources", "", certifications, destination, constants.DefaultCertificationsFolder, false, constants.Certifications).Return(nil)
+			getter.On("GetLocalResources", "", standards, destination, constants.DefaultStandardsFolder, false, constants.Standards).Return(nil)
+			getter.On("GetLocalResources", "", components, destination, constants.DefaultComponentsFolder, true, constants.Components).Return(nil)
+			getter.On("GetRemoteResources", destination, constants.DefaultCertificationsFolder, dependentCertifications).Return(nil)
+			getter.On("GetRemoteResources", destination, constants.DefaultStandardsFolder, dependentStandards).Return(expectedError)
+		})
+		It("should return an error when it's unable to get remote components", func() {
+			expectedError = errors.New("Remote components error")
+			getter.On("GetLocalResources", "", certifications, destination, constants.DefaultCertificationsFolder, false, constants.Certifications).Return(nil)
+			getter.On("GetLocalResources", "", standards, destination, constants.DefaultStandardsFolder, false, constants.Standards).Return(nil)
+			getter.On("GetLocalResources", "", components, destination, constants.DefaultComponentsFolder, true, constants.Components).Return(nil)
+			getter.On("GetRemoteResources", destination, constants.DefaultCertificationsFolder, dependentCertifications).Return(nil)
+			getter.On("GetRemoteResources", destination, constants.DefaultStandardsFolder, dependentStandards).Return(nil)
+			getter.On("GetRemoteResources", destination, constants.DefaultComponentsFolder, dependentStandards).Return(expectedError)
+		})
+		It("should return no error when able to get all components", func() {
+			expectedError = nil
+			getter.On("GetLocalResources", "", certifications, destination, constants.DefaultCertificationsFolder, false, constants.Certifications).Return(nil)
+			getter.On("GetLocalResources", "", standards, destination, constants.DefaultStandardsFolder, false, constants.Standards).Return(nil)
+			getter.On("GetLocalResources", "", components, destination, constants.DefaultComponentsFolder, true, constants.Components).Return(nil)
+			getter.On("GetRemoteResources", destination, constants.DefaultCertificationsFolder, dependentCertifications).Return(nil)
+			getter.On("GetRemoteResources", destination, constants.DefaultStandardsFolder, dependentStandards).Return(nil)
+			getter.On("GetRemoteResources", destination, constants.DefaultComponentsFolder, dependentStandards).Return(nil)
+		})
+		AfterEach(func() {
+			err := GetResources("", destination, s, getter)
+			assert.Equal(GinkgoT(), expectedError, err)
+			getter.AssertExpectations(GinkgoT())
+		})
```

Comment:
unfortunately no due to this assertion `getter.AssertExpectations(GinkgoT())`

if i want to set the return values for the 3 calls to `GetLocalResources` and the 3 calls to `GetRemoteResources`, i need to make sure all 6 calls will be hit currently due to that assertion which checks to make sure your mocks were actually called. 

for example, the first case, i return an error on the very first call.
```go
		It("should return an error when it's unable to get local certifications", func() {
			expectedError = errors.New("Cert error")
			getter.On("GetLocalResources", "", certifications, destination, constants.DefaultCertificationsFolder, false, constants.Certifications).Return(expectedError)
		})
```
i can't add the other 5 calls because since the first one returns an error, the other functions aren't called and when that assertion is run, it will fail.